save-core/src/commonMain/kotlin/org/cqfn/save/core/files/ConfigDetector.kt

```diff
@@ -0,0 +1,79 @@
+package org.cqfn.save.core.files
+
+import org.cqfn.save.core.config.TestSuiteConfig
+import org.cqfn.save.core.config.isDefaultSaveConfig
+
+import okio.FileSystem
+import okio.Path
+
+/**
+ * A class that is capable of discovering config files hierarchy.
+ */
+class ConfigDetector {
```

Comment:
why do we need class here... ?

Reply:
In case we decide to enhance this logic in the future, mainly. This logic could be added to the `TestSuiteConfig` itself, but I decided it would look better in a separate class (and look weird as a bunch of top-level functions)

Reply:
as you wish :)